websites/C/Custom Status/dist/metadata.json

```
@@ -6,7 +6,7 @@
 	},
 	"service": "Custom Status",
 	"description": {
-		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything but using custom images. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using!",
+		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything except the name. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using it!",
 		"tr": "Custom Status, Discord'un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak istediğiniz resmi seçme olanağı dışında bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) sayfaya gitmelisiniz.",
```

@eggsy can we have a translation here?

---

Eggsy told me to do this change so i don't think he can.

---

EGGSY sent this to me
```suggestion
"tr": "Custom Status, Discord’un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) adrese gitmelisiniz.",
```